packages/app-btc/src/operations/signTxn/index.ts

```diff
@@ -62,52 +62,50 @@ export const signTxn = async (
     meta: {
       version: signTxnDefaultParams.version,
       locktime: params.txn.locktime ?? signTxnDefaultParams.locktime,
-      inputSize: params.txn.inputs.length,
-      outputSize: params.txn.inputs.length,
-      hashType: params.txn.hashType ?? signTxnDefaultParams.hashtype,
+      inputCount: params.txn.inputs.length,
+      outputCount: params.txn.outputs.length,
+      sighash: params.txn.hashType ?? signTxnDefaultParams.hashtype,
     },
   });
+  const { metaAccepted } = await helper.waitForResult();
+  assertOrThrowInvalidResult(metaAccepted);
 
   for (let i = 0; i < params.txn.inputs.length; i += 1) {
-    const { input: inputRequest } = await helper.waitForResult();
-    assertOrThrowInvalidResult(inputRequest);
-    assertOrThrowInvalidResult(inputRequest.index === i);
-
     const input = params.txn.inputs[i];
     await helper.sendQuery({
       input: {
-        prevTxn: hexToUint8Array(input.prevTxn),
         prevTxnHash: hexToUint8Array(input.prevTxnHash),
-        prevIndex: input.prevIndex,
+        prevOutputIndex: input.prevIndex,
         scriptPubKey: hexToUint8Array(input.scriptPubKey),
         value: input.value,
         sequence: input.sequence ?? signTxnDefaultParams.input.sequence,
-        chainIndex: input.chainIndex,
+        changeIndex: input.chainIndex,
```

Comment:
Rename the params to `changeIndex` as well
```suggestion
        changeIndex: input.changeIndex,
```